integreat_cms/cms/migrations/0078_alter_poi_category.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.20 on 2023-07-25 15:46
+
+from django.db import migrations, models
+
+from ..constants import poicategory
+
+
+# pylint: disable=unused-argument
+def check_and_update_poicategory(apps, schema_editor):
+    """
+    This function checks whether the default POI category "Other" exists and cretes it if not.
+    It checks furthermore whether each location has a category and assigns the default category if not.
+    """
```

Comment:
```suggestion
    """
    Language = apps.get_model("cms", "Language")
    default_language = Language.objects.first()
    if not default_language:
        # If no languages exists, we can assume that also no POIs exist
        return

```
> To create a translation for the newly generated other category (if any does not exists), there is a difficulty: no language exists at the moment of reading this migration but POITranslation model needs one...... Of course there is at least one language in the real database, it won't work in the development environment. And creating a language object in the migration may bring a conflict with testdata or language objects in the real database......

Ah ok, but I'd say that if not language object exists, then we probably also don't have  POI object, since we cannot create a POI objects without a POITranslation and no POITranslation without a Language :grin:
So I'd argue we can assume the database is at least in a consistent state before running this migration (and just skip the function if no language exists)...